### DISCUSSION TITLE
【小程序问题】 这里是有关小程序问题的搜集

### DISCUSSION THREAD

**Discussion by waterbang** [Moonofweisheng/wot-design-uni#751] [General]
1.  小程序input 熟悉 `placeholder-style` 模拟器没有效果，真机有效。

**Comment by wangshijun0012** [comment]
const swiperList = ref([
	'https://registry.npmmirror.com/wot-design-uni-assets/1.0.4/files/capybara.jpg',
	'https://previewengine.zoho.com.cn/image/WD/jkblv66599ddbe13b4eb1aab3b6cb1792e295'
])

使用swiper组件，第一张图片显示，第二张不显示(network里面只对第一张图片发起了请求)。使用image标签src为第二张图片，正常显示。可以确定就是swiper的问题。何解？

**Comment by Moonofweisheng** [comment]
指定type为image



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年12月12日(周四) 下午4:17
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: Re: [Moonofweisheng/wot-design-uni] 【小程序问题】 这里是有关小程序问题的搜集 (Discussion #751)




 
const swiperList = ref([
 'https://registry.npmmirror.com/wot-design-uni-assets/1.0.4/files/capybara.jpg',
 'https://previewengine.zoho.com.cn/image/WD/jkblv66599ddbe13b4eb1aab3b6cb1792e295'
 ])
 
使用swiper组件，第一张图片显示，第二张不显示(network里面只对第一张图片发起了请求)。使用标签src为第二张图片，正常显示。可以确定就是swiper的问题。何解？

**Reply by wangshijun0012** [reply]
好了，感谢
